validator/src/main/java/com/amazon/aoc/App.java

```diff
@@ -138,6 +138,15 @@ public class App implements Callable<Integer> {
       defaultValue = "")
   private String instanceAmi;
 
+  @CommandLine.Option(
+          names = {"--asg-name"},
+          defaultValue = "")
+  private String asgName;
```

Comment:
Do we need this? We should use the one we use for cluster which is `--platform-info`